node-js-basics-101/node_modules/xmlhttprequest-ssl/example/demo.js

```diff
@@ -0,0 +1,16 @@
+var sys = require('util');
+var XMLHttpRequest = require("xmlhttprequest").XMLHttpRequest;
```

Comment:
__[jshint]__
* Redefinition of 'XMLHttpRequest'.

<!-- Comment by [SideCI](https://sideci.com) -->